bitnami/odoo/16/debian-12/rootfs/opt/bitnami/scripts/odoo/bitnami-templates/odoo.conf.tpl

```diff
@@ -22,7 +22,7 @@ http_port = {{ODOO_PORT_NUMBER}}
 ; https://www.odoo.com/forum/help-1/question/cpu-time-limit-exceeded-how-to-solve-it-87922
 limit_time_cpu = 90
 limit_time_real = 150
-; list_db = True
+list_db = {{#if ODOO_LIST_DB}}True{{else}}False{{/if}}
```

Comment:
`render-template` is not compatible with the "yes"/"no" values of `ODOO_LIST_DB`, leading to unexpected values for list_db.

Alternatively, I suggest the following:

1.

```suggestion
list_db = {{list_db}}
```

2. In `libodoo.sh`:

```diff
-        render-template "${template_dir}/odoo.conf.tpl" > "$ODOO_CONF_FILE"
+        list_db="$(is_boolean_yes "$ODOO_LIST_DB" && echo 'True' || echo 'False')" \
            render-template "${template_dir}/odoo.conf.tpl" > "$ODOO_CONF_FILE"
```

What do you think? Please, if you agree, apply the suggestion to the 3 branches 15, 16 and 17.

Reply:
Makes total sense. Applying.